docs/src/Motivation.md

```diff
@@ -0,0 +1,5 @@
+NoteMate is a programmatic transformation tool used to parse and manipulate Common Markdown files whose heading structure and contents follow [the Open Knowledge Model](https://jacobzelko.com/04172022033744-open-knowledge-standard/index.html). Due to the way NoteMate internally represents documents, it is however possible to transform notes from different structures to the Open Knowledge Model standard, and to express the standard in different output formats. 
+
+The Open Knowledge Model is well-suited for storing and presenting information in a natively human-readable and -workable format without dependency on third-party metadata tools, following academic inspirations for citations and sourcing information. This makes it a good choice for building digital gardens that can be grown with a high-quality, low-tech evidence chain.
+
+The strong suite of NoteMate is bringing this from the local storage in plain markdown to an intermediate structure you can then translate into various other formats suitable for web hosting or other plattforms.
```

Comment:
One thing is that even though NoteMate is geared primarily to having a parser only for Markdown, other parsers could be written for other files which is why I am removing the language about markdown specifically. 

Otherwise, I really like the notion of digital garden management. Could you maybe rewrite it such that digital gardens are highlighted in the beginning? I think that is a compelling idea for sure.

Reply:
Hmmmm. I wrote this in part because I wanted to capture the "is" state and origin of the module while still keeping the immediate premise in scope, and without going too far into what it *could* be while loosing clarity about what it can do, right now. 

A list of "current features" or "capabilities" in the documentation could help here, but even so, this is a gradient.